proposals/lip-0066.md

```diff
@@ -45,6 +45,16 @@ We define the following constants:
 | `PrivateKeyEd25519` | bytes  | Byte sequences of length `ED25519_PRIVATE_KEY_LENGTH`. | An Ed25519 private key. |
 | `PrivateKeyBLS`     | bytes  | Byte sequences of length `BLS_PRIVATE_KEY_LENGTH`. | An BLS private key. |
 
+## bytesToInt Conversion
```

Comment:
```suggestion
## uint32be
```

Reply:
In the past we used this function name to convert bytes to uint

Reply:
Replaced.